### PR TITLE
django_polymorphic

### PR DESCRIPTION
**Pull request by ocefpaf**
Last missing piece of #87

Ping @kwilcox

### CODE REVIEW

**Comment by kwilcox** [comment]
I think the tests will fail with no Django context?

**Comment by ocefpaf** [comment]
The latest version is smarter about this.

File: django_polymorphic/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        - python
+        - django
+
+test:
+    imports:
+        - polymorphic
+
+about:
+    home: https://github.com/chrisglass/django_polymorphic
+    license: BSD-3-Clause
```
Comment (jakirkham) [inline django_polymorphic/meta.yaml]: Could we drop the first `-` so it would be `BSD 3-Clause`?

**Comment by jakirkham** [comment]
Good thing you solved underscores, @ocefpaf. :smile:

File: django_polymorphic/meta.yaml (hunk 1 above)
Comment (ocefpaf) [inline django_polymorphic/meta.yaml]: I don't think we should be that strict. The information needed is there...

Comment (jakirkham) [inline django_polymorphic/meta.yaml]: Mainly I was being strict as I figure the logic proposed to the linter will prefer this sort of thing.

Comment (ocefpaf) [inline django_polymorphic/meta.yaml]: I know. But I have to revisit that idea in light of https://github.com/conda-forge/conda-smithy/pull/93#issuecomment-201733789

Comment (jakirkham) [inline django_polymorphic/meta.yaml]: Ah, right, I forgot. Sorry. Thanks for jogging my memory.

**Comment by jakirkham** [comment]
Thanks @ocefpaf.

File: django_polymorphic/meta.yaml (hunk 1 above)
Comment (ocefpaf) [inline django_polymorphic/meta.yaml]: > Ah, right, I forgot. Sorry. Thanks for jogging my memory.

If you can remember everything that is going on in conda-forge you are a robot! :wink:

**Comment by ocefpaf** [comment]
@jakirkham We both did not notice, but this was not in the recipes directory and the feedstock was not created :unamused:

**Comment by jakirkham** [comment]
Ouch! How did this happen?

**Comment by ocefpaf** [comment]
I was :zzz: :wink:

**Comment by jakirkham** [comment]
My notifications are very fast. Also, they don't intermingle with my email. :racehorse:

**Comment by jakirkham** [comment]
Did you see [this in the log]( https://travis-ci.org/conda-forge/staged-recipes/jobs/119950137#L241 )? Appears there was nothing to commit. Pretty weird.

**Comment by ocefpaf** [comment]
[This one](https://circleci.com/gh/conda-forge/staged-recipes/1236) is the one that matters now :wink:

File: django_polymorphic/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline django_polymorphic/meta.yaml]: lol, your hilarious, @pelson.

**Pull request merged**